From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Tim Mullin <tim@cpanel.net>
Date: Thu, 9 May 2019 21:14:37 -0500
Subject: [PATCH] Ensure Phalcon is built so that it supports older CPUs.

---
 build/install | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/build/install b/build/install
index 0e5f896..580adb8 100755
--- a/build/install
+++ b/build/install
@@ -24,7 +24,7 @@
 
 # Check best compilation flags for GCC
 export CC="gcc"
-export CFLAGS="-march=native -mtune=native -O2 -fomit-frame-pointer"
+export CFLAGS="-O2 -fomit-frame-pointer"
 export CPPFLAGS="-DPHALCON_RELEASE"
 
 # Set defaults
@@ -79,9 +79,7 @@ fi
 echo "int main() {}" > t.c
 gcc ${CFLAGS} t.c -o t 2> t.t
 if [ $? != 0 ]; then
-	chmod +x gcccpuopt
-	BFLAGS=`./gcccpuopt`
-	export CFLAGS="-O2 -fomit-frame-pointer $BFLAGS"
+	export CFLAGS="-O2 -fomit-frame-pointer"
 	gcc ${CFLAGS} t.c -o t 2> t.t
 	if [ $? != 0 ]; then
 		export CFLAGS="-O2"
